### PR TITLE
[Bots][Reviewandupdatecontent][1427059]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Reviewed document for grammar and language issues. Please review.

### CODE REVIEW

File: msteams-platform/bots/bot-basics.md [hunk 1]
```diff
@@ -43,159 +43,164 @@ To implement your logic for Teams specific activity handlers, you need to overri
...
+| signin/verifyState              | `OnTeamsSigninVerifyStateAsync`      | This method is invoked when a signIn verify state activity is received from the connector. |
+| task/fetch                      | `OnTeamsTaskModuleFetchAsync`        | This method can be overridden in a derived class to provide logic when a task module is fetched. |
+| task/submit                     | `OnTeamsTaskModuleSubmitAsync`       | This method can be overridden in a derived class to provide logic when a task module is submitted. |
 
-The invoke activities listed above are for conversational bots in Teams. The Bot Framework SDK also supports invokes specific to messaging extensions. For more information see [What are messaging extensions](https://aka.ms/azure-bot-what-are-messaging-extensions)
+The invoke activities listed in this section are for conversational bots in Teams. The Bot Framework SDK also supports invoke activities specific to messaging extensions. For more information see [What are messaging extensions](https://aka.ms/azure-bot-what-are-messaging-extensions).
```
Comment (KirtiPereira) [inline msteams-platform/bots/bot-basics.md:99]:
```Suggestion
The invoke activities listed in this section are for conversational bots in Teams. The Bot Framework SDK also supports invoke activities specific to messaging extensions. For more information, see [What are messaging extensions](https://aka.ms/azure-bot-what-are-messaging-extensions).
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - Have shared my comments. Please verify and commit the suggestions.
Thank you.

**Review by Manish-MSFT — APPROVED** [review]
Everything looks good, no changes from my side.

**Review by KirtiPereira — APPROVED** [review]
Approved for merging.

**Review by surbhigupta12 — APPROVED** [review]
Approving this PR.

**Pull request merged**